ecc_enclave/enclave/enclave.cpp

```diff
@@ -224,6 +228,23 @@ int ecall_cc_invoke(const char* encoded_args,
 {
     LOG_DEBUG("ecall_cc_invoke: \tArgs: %s", encoded_args);
 
+    // NOTE/TODO: this crypto part will be removed
+    if(!crypto_created)
+    {
+        pdo::crypto::sig::PublicKey verification_key_;
+        pdo::crypto::sig::PrivateKey signature_key_;
+        signature_key_.Generate(); //private key
+        verification_key_ = signature_key_.GetPublicKey();
+        //debug
+        std::string s = verification_key_.Serialize();
+        LOG_DEBUG("enclave verification key: %s", s.c_str());
```

Comment:
Have you actually ever checked if this serialised key can be used plain go crypto? We want to use the pdo crypto lib to have only a single crypto implementation used everywhere inside the enclave and outside using some wrapper in go. However, we haven't really checked with the fabric community if this is fine OR if we insist of a go implementation where ever possible.

Reply:
In particular, we should double check that we can use this crypto lib in our custom ecc-vscc.

Reply:
> Have you actually ever checked if this serialised key can be used plain go crypto? We want to use the pdo crypto lib to have only a single crypto implementation used everywhere inside the enclave and outside using some wrapper in go. However, we haven't really checked with the fabric community if this is fine OR if they insist of a go implementation where ever possible.

I have not tested, but my answer is yes. The serialized key is in PEM format: `-----BEGIN Type-----Headers\nbase64-encoded Bytes-----END Type-----`. So you can use the `encoding/pem` package to encode/decode them, and then use `crypto`.

> In particular, we should double check that we can use this crypto lib in our custom ecc-vscc.

As the plugin will not verify the attestation, we could simply resort to crypto to verify the signature.